TeamCode/src/main/java/org/firstinspires/ftc/teamcode/movement/Power.java

```diff
@@ -0,0 +1,37 @@
+package org.firstinspires.ftc.teamcode.movement;
+
+public class Power {
```

Comment:
Ori folosesti `this` peste tot, ori deloc.

Reply:
Sunt obligat sa folosesc `this` in snippetul ```
this.fl = fl;
this.fr = fr;
this.bl = bl;
this.br = br;``` deoarece numele fieldurilor clasei si parametrilor constructorilor coincid.